charts/karavi-observability/templates/otel-collector.yaml

```diff
@@ -26,7 +26,6 @@ metadata:
     app.kubernetes.io/name: otel-collector
     app.kubernetes.io/instance: {{ .Release.Name }}
 spec:
-  type: NodePort
```

Comment:
This can be made configurable similar to topology (https://github.com/dell/helm-charts/blob/main/charts/karavi-observability/templates/karavi-topology.yaml#L9) for cases such as e2e where we need to deploy as NodePort.

Reply:
Updated